lib/equipment_import.rb

```diff
@@ -96,7 +96,7 @@ def import_objects(processed_objects) # rubocop:disable MethodLength
         EquipmentModel.where('name = ?', object_data[:equipment_model]).first
 
       # create new category
-      object = EquipmentObject.new(object_data)
+      object = EquipmentItem.new(object_data)
```

Comment:
we should update these variable names (there are probably a bunch in this file).